frontend/src/metabase-lib/join.ts

```diff
@@ -196,12 +196,12 @@ export function joinConditionOperators(
   return ML.join_condition_operators(query, stageIndex, lhsColumn, rhsColumn);
 }
 
-export function suggestedJoinCondition(
+export function suggestedJoinConditions(
   query: Query,
   stageIndex: number,
   joinable: Joinable,
-): JoinCondition | null {
-  return ML.suggested_join_condition(query, stageIndex, joinable);
+): JoinCondition[] | null {
```

Comment:
Wdyt about returning an empty array when there are no suggested conditions and not null?

Reply:
done